apps/desktop-wallet/src/api/apiDataHooks/address/useFetchAddressBalancesAlph.ts

```diff
@@ -30,22 +30,21 @@ interface UseAddressAlphBalancesProps extends SkipProp {
 
 const useFetchAddressBalancesAlph = ({ addressHash, skip }: UseAddressAlphBalancesProps) => {
   const networkId = useAppSelector((s) => s.network.settings.networkId)
-  const queryProps = { addressHash, networkId }
-
-  const { data: txHashes, isLoading: isLoadingTxHashes } = useQuery(addressLatestTransactionHashQuery(queryProps))
 
+  const { data: updatesSignal, isLoading: isLoadingUpdatesSignal } = useFetchAddressUpdatesSignal({ addressHash, skip })
```

Comment:
I decided to rename `useFetchAddressLatestTransaction` to something that is more accurate based on its usage. It's used as a "signal" to whether all other queries should invalidate or not.

Reply:
> With the precedent name I knew that it returned a transaction.

In fact, it is not returning a transaction. It was returning an object:

```ts
{
   latestTx: Transaction | undefined
   previousTx: Transaction | undefined
}
```

I wanted to find a name that is more appropriate for its usage, not for what it returns. The "signal" that we have outdated data is the latest transaction hash. I'm happy to discuss a different naming.